store/dataStore.ts

```diff
@@ -0,0 +1,17 @@
+import { create } from 'zustand';
+
+interface Example {
+    id: number;
+    name: string;
+}
+interface ExampleStore {
+    examples: Example[];
+}
+
+const dataStore = () => ({
+    examples: [],
```

Comment:
Just curious, are we also setting example to set the base for an example? ie. `setExample`

Reply:
@choir27 Just to clarify, are you referring to use `set` ?

Reply:
Yup!

Reply:
As of the moment, I'm not using `set` since the objective for task #201 is to add a base.